### DISCUSSION TITLE
Best way to use multiple jsonl files

### DISCUSSION THREAD

**Discussion by Asbjoern** [HASwitchPlate/openHASP#437] [Documentation]
What is the best/intended way to split up large jsonl files in multiple and have them all load?
Is there a kind of _import_ function to use in pages.jsonl?

The main reason for splitting up is to reuse pages in several plates and put them under version control. 
The [openhasp-config-manager](https://github.com/markusressel/openhasp-config-manager) project generates several jsonl files but it is not clear to me the intended way to load them.

For now I am using boot.cmd:
```
run /file1.jsonl
run /file2.jsonl
run /file3.jsonl
```
But it does not feel like the right way.

Any help?

Asbjørn

**Comment by fvanroie** [comment]
That's how I would do it too. Why doesn't that feel right?

**Comment by Asbjoern** [comment]
I think I just had my mind locked the pages.jsonl as a main file and expected some kind of import function to be used in it.
I am glad to hear the cmd way is the intended way.
It might be a good idea to mention this somewhere in the documentation or even an example for splitting up in multiple pages and dynamically load and delete pages and objects, eg. from button action.
